spec/puppetserver/ca/action/list_spec.rb

```diff
@@ -66,5 +66,26 @@
       expect(out.string).to match(/Signed Certificates:.*foo.*\(SHA256\).*three.*alt names:.*"DNS:foo", "DNS:bar".*/m)
       expect(out.string).to match(/Missing Certificates:.*fake.*/m)
     end
+
+    it 'errors when unknown format option is passed in' do
+      allow(action).to receive(:get_all_certs).and_return(result)
+      exit_code = action.run({'certname' => ['foo', 'baz'], 'format' => 'text'})
```

Comment:
We want to allow 'text', right?

Reply:
Good eyes, I was goofing around with the test and did that only for json.